test/src/unittests/tonal/test_chordsdetectionbeats.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2006-2022  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+from essentia_test import *
+import numpy as np
+
+
+class TestChordsDetectionBeats(TestCase):
+    def testEmpty(self):
+        with self.assertRaises(EssentiaException):
+            ChordsDetectionBeats()(np.zeros(shape=(0, 12), dtype=np.single), np.zeros(shape=(0,), dtype=np.single))
```

Comment:
There are several options here we can check:
- empty input `pcp` and `ticks` (tested now)
- empty input `pcp` but non-empty `ticks`
- empty input `ticks` but non-empty `pcp`

Also, from the DOC description of the algorithm it is not clear what are the requirements on the input `ticks`. Let's add this information to the DOC string. What happens if there is a single `ticks` value or no values?